src/DynamoCoreWpf/ViewModels/Watch3D/DefaultWatch3DViewModel.cs

```diff
@@ -95,7 +95,7 @@ public bool Active
 
                 OnActiveStateChanged();
 
-                if (active)
+                if (active && this is HelixWatch3DViewModel)
```

Comment:
In such cases where you have derived classes needing to override behavior in the base class, it's better design to override the method/property in the derived class and let polymorphism do its job. So in this case it would be overriding `Active` in `HelixWatch3DViewModel`.